project/SparkBuild.scala

```diff
@@ -765,16 +750,6 @@ object CopyDependencies {
 
 }
 
-object Java8TestSettings {
-  import BuildCommons._
-
-  lazy val settings = Seq(
-    javacJVMVersion := "1.8",
-    // Targeting Java 8 bytecode is only supported in Scala 2.11.4 and higher:
-    scalacJVMVersion := (if (System.getProperty("scala-2.10") == "true") "1.7" else "1.8")
```

Comment:
The actually `scalac` I'd be running locally would be 2.12, as that's what I have installed. I'm guessing it's fine cross-compiling against a 2.10 library, fine enforcing a 2.10 language level, but also fine emitting 1.8 byte code?